opensrp-tasking/src/main/java/org/smartregister/tasking/configuration/DefaultTaskingLibraryConfiguration.java

```diff
@@ -0,0 +1,439 @@
+package org.smartregister.tasking.configuration;
+
+import android.app.Activity;
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.util.Pair;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.geojson.FeatureCollection;
+
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.json.JSONObject;
+import org.smartregister.clientandeventmodel.Event;
+import org.smartregister.commonregistry.CommonPersonObjectClient;
+import org.smartregister.domain.Location;
+import org.smartregister.domain.Task;
+import org.smartregister.tasking.TaskingLibrary;
+import org.smartregister.tasking.activity.TaskingHomeActivity;
+import org.smartregister.tasking.adapter.TaskRegisterAdapter;
+import org.smartregister.tasking.contract.BaseContract;
+import org.smartregister.tasking.contract.BaseDrawerContract;
+import org.smartregister.tasking.contract.BaseFormFragmentContract;
+import org.smartregister.tasking.contract.TaskingHomeActivityContract;
+import org.smartregister.tasking.layer.DigitalGlobeLayer;
+import org.smartregister.tasking.model.BaseTaskDetails;
+import org.smartregister.tasking.model.CardDetails;
+import org.smartregister.tasking.model.TaskDetails;
+import org.smartregister.tasking.model.TaskFilterParams;
+import org.smartregister.tasking.repository.TaskingMappingHelper;
+import org.smartregister.tasking.util.ActivityConfiguration;
+import org.smartregister.tasking.util.DefaultLocationHierarchyUtils;
+import org.smartregister.tasking.util.GeoJsonUtils;
+import org.smartregister.tasking.util.TaskingConstants;
+import org.smartregister.tasking.util.TaskingJsonFormUtils;
+import org.smartregister.tasking.util.TaskingLibraryConfiguration;
+import org.smartregister.tasking.util.TaskingMapHelper;
+import org.smartregister.tasking.view.DrawerMenuView;
+import org.smartregister.tasking.viewholder.TaskRegisterViewHolder;
+import org.smartregister.util.AppExecutors;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.smartregister.tasking.util.Utils.getGlobalConfig;
+
+public class DefaultTaskingLibraryConfiguration extends TaskingLibraryConfiguration {
+
+    private boolean isSynced;
+
+    @NonNull
+    @Override
+    public Pair<Drawable, String> getActionDrawable(Context context, TaskDetails task) {
+        return null;
```

Comment:
Can we prevent NPEs and any further customization of the `DefaultTaskingLibraryConfiguration` by providing safe default values?